### ISSUE TITLE
Textual indicator

### ISSUE THREAD

**Issue by guadalupebabio** [CityScope/CS_Brix#37]
Can we add an indicator that interacts with the textual layer that Ariel just created in [this issue](https://github.com/CityScope/CS_cityscopeJS/issues/119)? 

The frontend will read from this endpoint:
```
http://cityio.media.mit.edu/api/table/lomas/TEXTUAL
```

And the data has to be formatted as:
```
[{
 	"id": 450,
 	"info": "yes!"
 },{
 	"id": 40,
 	"info": "no!"
 }]
```

**Comment by crisjf** [comment]
@guadalupebabio I just implemented it but haven't had the chance to write the docs. 

To use it with your hybrid indicator, define a function called `return_indicator_textual`, similarly to how you work with `return_indicator_heatmap` and `return_indicator_numeric`. Make sure your function has the right format.

Alternatively, if you want to define your own `return_indicator` for your hybrid indicator, make sure you return an object of the form:
```
out = {
	'heatmap': heatmap_values,
	'numeric': numeric_values,
	'textual': textual_values
}
```
Where `textual_values` is a list of dicts with two keys.

**Comment by crisjf** [comment]
Documentation is missing for this feature.

**Comment by crisjf** [comment]
I added an example of a textual indicator in the examples section and added an explanation in the tutorial as well. 

https://cityscope.media.mit.edu/CS_Brix/tutorial.html
https://cityscope.media.mit.edu/CS_Brix/examples.html#textual-indicator

**Comment by crisjf** [comment]
Please note there was a bug that made the Handler not accept textual indicators in `v0.0.9`. This was fixed in ca9176fe02de6bd7a71fc98b9237212ca17fcebb but has not yet been updated in pip.
